src/App.jsx

```diff
@@ -1,7 +1,14 @@
 import React from 'react';
+import { Button, Typography } from '@mui/material';
 
 function App() {
-	return <h1>Ivad Code</h1>;
+	return (
+		<div>
+			<Typography>Ivad code</Typography>
+
```

Comment:
Reduce unused spaces... but no bigg

Reply:
I think that's an issue with GitHub. let me confirm that prettier is working as expected

Reply:
@Abdulkeza prettier is working as expected and my code are well-formatted locally definitely the issues with Github